docs/examples/Asymmetric Calculation Example.ipynb

```diff
@@ -0,0 +1,828 @@
+    "# Asymmetric Calculation Example\n",
+    "\n",
+    "In this notebook we will walk through an example of asymmetric calculation using `power-grid-model`. \n",
+    "The following points are covered\n",
+    " * Construction of the model\n",
+    " * Run asymmetric power flow calculation once, and its relevant function arguments\n",
+    " * Run asymmetric power flow in batch calculations, and its relevant function arguments\n",
+    " * Run state estimation once, and its relevant function arguments\n",
+    "\n",
+    "This notebook serves as an example of how to use the Python API. For detailed API documentation, refer to\n",
+    "[Python API reference](https://power-grid-model.readthedocs.io/en/stable/api_reference/python-api-reference.html)\n",
+    "and [Native Data Interface](https://power-grid-model.readthedocs.io/en/stable/advanced_documentation/native-data-interface.html).\n",
+    "\n",
+    "\n",
+    "## Example Network\n",
+    "\n",
+    "We use a simple network with 3 nodes, 1 source, 3 lines, and 2 loads (1 symmetric load, 1 asymmetric load). As shown below:\n",
+    "\n",
+    "```\n",
+    " -----------------------line_8---------------\n",
+    " |                                          |\n",
+    "node_1 ---line_3--- node_2 ----line_5---- node_6\n",
+    " |                    |                     |\n",
+    "source_10          sym_load_4           asym_load_7\n",
+    "```\n",
+    "\n",
+    "The 3 nodes are connected in a triangular way by 3 lines.\n",
+    "\n",
+    "**NOTE: load_7 is asymmetric in this case.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ae11dc9a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# some basic imports\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "from power_grid_model import LoadGenType\n",
+    "from power_grid_model import PowerGridModel, CalculationMethod, CalculationType, MeasuredTerminalType\n",
+    "from power_grid_model import initialize_array"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "f983cef7",
+   "metadata": {},
+   "source": [
+    "## Power Flow Calculation\n",
+    "### Input Dataset\n",
+    "\n",
+    "We create input dataset by using the helper function `initialize_array`.\n",
+    "Note the units of all input are standard SI unit without any prefix,\n",
+    "i.e. the unit of voltage is volt (V), not kV.\n",
+    "\n",
+    "Please refer [Components](https://power-grid-model.readthedocs.io/en/stable/user_manual/components.html) for detailed explanation of all component types and their input/output attributes. \n",
+    "\n",
+    "**NOTE: The required attributes of each components can be different for asymmetric calculations.**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6f008736",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# node\n",
+    "node = initialize_array(\"input\", \"node\", 3)\n",
```

Comment:
It would be nice to include a param explanation in a short inline comment, e.g.:
`node =  initialize_array("input", "node", 3) # data_type, component_type, shape `

This was not done in other examples so just take it with grain of salt ;)

Reply:
Not really, we refer to the documentation.